src/Chapter16/Listing16.11.SelectingTupleFollowingTheGroupbyClause.cs

```diff
@@ -1,45 +1,54 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter16.Listing16_11
 {
     using AddisonWesley.Michaelis.EssentialCSharp.Chapter16.Listing16;
+    #region INCLUDE
     using System;
     using System.Collections.Generic;
     using System.Linq;
-
+    #region EXCLUDE
     public class Program
     {
         public static void Main()
         {
             GroupKeywords1();
         }
-
+        #endregion EXCLUDE
         private static void GroupKeywords1()
         {
+            #region HIGHLIGHT
             IEnumerable<IGrouping<bool, string>> keywordGroups =
                 from word in CSharp.Keywords
                 group word by word.Contains('*');
 
-            IEnumerable<(bool IsContextualKeyword, IGrouping<bool, string> Items)> selection =
+            IEnumerable<
+                (bool IsContextualKeyword, IGrouping<bool, string> Items)> 
+                    selection =
```

Comment:
```suggestion
            IEnumerable<(bool IsContextualKeyword, 
                IGrouping<bool, string> Items)> selection =
```
I think this is a better split point